.github/workflows/jest.yml

```diff
@@ -1,4 +1,4 @@
-Name: 'Jest'
```

Comment:
this was preventing unit tests from running in the CI/CD pipeline up to this point 😁